lib/reek/cli/reek_command.rb

```diff
@@ -12,8 +12,8 @@ def execute(app)
         options.sources.each do |source|
           reporter.add_examiner Examiner.new(source, smell_names, configuration: app.configuration)
         end
-        reporter.smells? ? app.report_smells : app.report_success
         reporter.show
+        reporter.smells? ? options.failure_exit_code : options.success_exit_code
```

Comment:
While we are at it, let's rename "options" here to "options_interpreter"!
It's super confusing like it is right now and trips me up every time :-(